tests/test_harness.py

```diff
@@ -233,17 +235,15 @@ def test_no_live_logging(
         oss_artifact: Path,
     ) -> None:
         """
-        By default, subprocess output is not logged in the creation of a
-        cluster.
+        By default, subprocess output is not logged during DC/OS installation.
         """
         with pytest.raises(CalledProcessError):
-            # It is not possible to create a cluster with two master nodes.
             with Cluster(
                 masters=2,
                 cluster_backend=cluster_backend,
-                build_artifact=oss_artifact,
-            ):
-                pass  # pragma: no cover
+            ) as cluster:
+                # It is not possible to install DC/OS with two master nodes.
+                cluster.install_dcos_from_path(oss_artifact)
```

Comment:
Instead, put only this line in the `pytest.raises` context manager.

The advantage to this is that it is clearer what raised the error.